app/routes/theatre/routes.py

```diff
@@ -0,0 +1,18 @@
+"""PC est magique - Profile Pages Routes"""
+
+import flask
+from flask_babel import _
+
+from app import context, db
+from app.models import Spectacle, Representation
+from app.routes.theatre import bp
+from app.utils import helpers, typing
+
+
+@bp.route("/")
```

Comment:
```suggestion
@bp.route("")
@bp.route("/")
```

Petite subtilité : sans ça, https://pc-est-magique.fr/theatre ne renvoie pas sur cette route (il faut obligatoirement le / final)